torch/distributed/algorithms/quantization/quantization.py

```diff
@@ -85,6 +83,52 @@ def _dequantize_tensor_list(tensor_list, qtype, quant_loss=None):
     dequantized_tensor_list = [_dequantize_tensor(t, qtype) for t in tensor_list]
     return dequantized_tensor_list
 
+def _quantized_a2a_single_handler(*args, **kwargs):
+    group = kwargs.get('group', None)
+    async_op = kwargs.get('async_op', False)
+    qtype = args[0]
+    tensors = args[1]
+    input_tensor = args[2]
+    out_splits = kwargs.get('out_splits', None)
+    in_splits = kwargs.get('in_splits', None)
+    quant_loss = kwargs.get('quant_loss', None)
+    input_tensors = _quantize_tensor(input_tensor, qtype)
+    out_tensors = _quantize_tensor(tensors, qtype)
+    dist.all_to_all_single(out_tensors, input_tensors, out_splits, in_splits, group=group)
```

Comment:
Maybe we can directly pass in kwargs here. Since `kwargs` is a dict, we can `pop` any keyword arguments that are not relevant to the distributed collective and then directly pass in the kwargs dict.